libs/orders/src/lib/components/order-list/order-edit-dialog.tsx

```diff
@@ -69,13 +69,13 @@ export const OrderEditDialog = ({
         <div>
           <p className={headerClassName}>{t(`Size`)}</p>
           <p>
-            {
+            {order.market && (
```

Comment:
question (non-blocking): if not having a market is a possibility would it be better to throw an error here? Feels like all orders should be tied to a market in some way and if not we should be reporting that so we can look into it